src/GoogleAnalytics.ts

```diff
@@ -3,14 +3,14 @@ export const trackEvent = (
 	timingVar: string,
 	timingLabel: string,
 ): void => {
-	const { ga, guardian } = window;
+	const { ga, guardian, performance } = window;
 	const trackerName: string | undefined =
 		guardian.config?.googleAnalytics?.trackers.editorial;
 
-	if (typeof ga !== 'function') {
+	if (typeof ga !== 'function' || typeof performance == 'undefined') {
```

Comment:
Open question: is this actually the desired behaviour? 

It seems like we have two options when `performance.window` isn't available:

- return early and omit to call to `window.ga` (this is what we do in this PR)
- call `window.ga` anyway with some dummy value for `timeSincePageLoad`

Any thoughts?

Reply:
[Polyfill.io does this](https://github.com/Financial-Times/polyfill-library/blob/master/polyfills/performance/now/polyfill.js):

```ts
var startTime = Date.now();

if (!window.performance) {
	window.performance = {};
}

window.performance.now = function () {
	return Date.now() - startTime;
};
```

I’m wondering if the fix would actually be simply to add this feature to [our polyfill](https://github.com/guardian/frontend/blob/main/static/src/javascripts/polyfill.io)?

MDN Docs: https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
Ahhhh interesting point

Reply:
This makes good sense to me. I'll raise a couple of PRs (in `DCR `and `frontend`) and close this one if we collectively decide to go down this route.